CV/Official_Announcement.md

```diff
@@ -0,0 +1,89 @@
+
+Để giúp một số thành viên của hội nắm bắt được kiến thức căn bản về AI/ML, đặc biệt là các bạn đang làm Software Engineer muốn học và tìm hiểu về lĩnh vực này, từ năm 2019, hội VJAI sẽ tổ chức thêm 1 hoạt động mới là các khoá học ngắn về từng chủ đề trong lĩnh vực AI. Chúng ta sẽ bắt đầu với khoá học về chủ đề `Computer Vision and Deep Learning for beginner`.
+
+## Mục đích của khoá học
+* Mang lại kiến thức cơ bản về Deep Learning trong lĩnh vực Computer Vision
+* Giúp học viên có cái nhìn toàn diện về những tiến bộ của Deep Learning trong Computer Vision
```

Comment:
"toàn diện" nghe hơi cao siêu quá ạ, "khái quát" thì em nghĩ sẽ dễ cho mình hơn.

Reply:
Fixed